docs/api/desktop-capturer.md

```diff
@@ -94,6 +138,13 @@ Returns `Promise<DesktopCapturerSource[]>` - Resolves with an array of [`Desktop
 **Note** Capturing the screen contents requires user consent on macOS 10.15 Catalina or higher,
 which can detected by [`systemPreferences.getMediaAccessStatus`].
 
+### `desktopCapturer.getWebContentsStream(options)`
```

Comment:
Maybe `getWebContentsStreamInfo`? It's not really a stream yet.

Reply:
Right

At some point I was considering returning a `DesktopCapturerSource`, but:
- the description of `DesktopCapturerSource.id` would not match this case (not "screen:XX" format, that's the "mediaId")
- don't know if it should return the "mediaId" in the `display_id` field or not, since that's different format
- `name`, `thumbnail`, `appIcon` are not provided
- At this point, the returned `id` is expected to be used, otherwise it will leak inside `DesktopStreamsRegistry`. This behavior is not similar to screen/application sources.

I don't know if `getWebContentsSource` would be a good name or more confusing.